src/built_in.rs

```diff
@@ -0,0 +1,15 @@
+pub fn sum(x: i64, y: i64) -> i64 {
+    x + y
+}
+
+pub fn sub(x: i64, y: i64) -> i64 {
+    x - y
+}
+
+pub fn mul(x: i64, y: i64) -> i64 {
+    x * y
+}
+
+pub fn div(x: i64, y: i64) -> i64 {
+    x / y
+}
```

Comment:
These should actually take vectors of `Expression`s and the `fold` we implemented should go here.